"restore":{"projectUniqueName":"/Users/stivencabrera/RiderProjects/cs12dotnet8/Chapter04/CallStackExceptionHandling/CallStackExceptionHandling.csproj","projectName":"CallStackExceptionHandling","projectPath":"/Users/stivencabrera/RiderProjects/cs12dotnet8/Chapter04/CallStackExceptionHandling/CallStackExceptionHandling.csproj","outputPath":"/Users/stivencabrera/RiderProjects/cs12dotnet8/Chapter04/CallStackExceptionHandling/obj/","projectStyle":"PackageReference","originalTargetFrameworks":["net8.0"],"sources":{"https://api.nuget.org/v3/index.json":{}},"frameworks":{"net8.0":{"targetAlias":"net8.0","projectReferences":{"/Users/stivencabrera/RiderProjects/cs12dotnet8/Chapter04/CallStackExceptionHandlingLib/CallStackExceptionHandlingLib.csproj":{"projectPath":"/Users/stivencabrera/RiderProjects/cs12dotnet8/Chapter04/CallStackExceptionHandlingLib/CallStackExceptionHandlingLib.csproj"}}}},"warningProperties":{"warnAsError":["NU1605"]}}"frameworks":{"net8.0":{"targetAlias":"net8.0","imports":["net461","net462","net47","net471","net472","net48","net481"],"assetTargetFallback":true,"warn":true,"frameworkReferences":{"Microsoft.NETCore.App":{"privateAssets":"all"}},"runtimeIdentifierGraphPath":"/Users/stivencabrera/.dotnet/sdk/8.0.203/PortableRuntimeIdentifierGraph.json"}}